From 59cad66da7d8e5c7adf921df6c37242589fbd9d1 Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Sat, 13 Apr 2019 18:35:09 +0200
Subject: [PATCH] configs: meson64: add Mender requirements

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>
---
 configs/odroid-c2_defconfig | 4 ++++
 include/configs/meson64.h   | 3 +++
 2 files changed, 7 insertions(+)

diff --git a/configs/odroid-c2_defconfig b/configs/odroid-c2_defconfig
index 1f5a52c57c..3e8967e29b 100644
--- a/configs/odroid-c2_defconfig
+++ b/configs/odroid-c2_defconfig
@@ -41,3 +41,7 @@ CONFIG_DEBUG_UART_ANNOUNCE=y
 CONFIG_DEBUG_UART_SKIP_INIT=y
 CONFIG_MESON_SERIAL=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_ENV_IS_IN_MMC=y
+CONFIG_ENV_OFFSET=0x800000
+CONFIG_ENV_OFFSET_REDUND=0x1000000
+CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index 736081277d..a4304813b6 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -46,6 +46,9 @@
 #define CONFIG_SYS_LOAD_ADDR		CONFIG_SYS_TEXT_BASE
 #define CONFIG_SYS_BOOTM_LEN		(64 << 20) /* 64 MiB */
 
+#define CONFIG_BOOTCOUNT_LIMIT
+#define CONFIG_BOOTCOUNT_ENV
+
 /* ROM USB boot support, auto-execute boot.scr at scriptaddr */
 #define BOOTENV_DEV_ROMUSB(devtypeu, devtypel, instance) \
 	"bootcmd_romusb=" \
-- 
2.29.2

